### ISSUE TITLE
Fails silently if masters node can't access the S3 ops bucket

### ISSUE THREAD

**Issue by rata** [kubernetes/kops#2344]
I've created the kops bucket in one organization and the cluster in another sub-organization, and I'm just playing with kops for the first time.

The issue is that the cluster (the master nodes, at least) don't have access to the S3 kops bucket created in a parent AWS organization. And the master node running nodeup, looking at syslog, is stucked at:

 ```
Apr 11 20:09:50 ip-172-20-90-76 nodeup[730]: W0411 20:09:50.865908     730 main.go:138] got error running nodeup (will retry in 30s): error loading Cluster "s3://<bucket-name>/non-mc-clusters/alpha.aws-us-east-1.<some-zone>/cluster.spec": Could not retrieve location for AWS bucket <bucket-name>
``` 

Justin very helpfully in slack told me to report the issue, as organizations is a nice to have, and can probably be checked easily this permission issue so it doesn't fail so silently.

**Comment by jbarreneche** [comment]
Hi, we already solved the problem.

The issue happened because the bucket permissions weren't correctly set to work with cross account permissions. We had followed the [AWS instructions for setting up the cross account bucket access](http://docs.aws.amazon.com/AmazonS3/latest/dev/example-walkthroughs-managing-access-example2.html) but instead of delegating from the *main account* to the `root` user on the *secondary account*, we granted the permissions to the role with "admin" in the secondary account.

We then used this role to create the kops state (without any issue), and thought the setup was going to work. But, because the root user from the *secondary account* didn't have permissions, it caused the policies that kops creates to don't have any effect.
